assets/main-search.js

```diff
@@ -0,0 +1,49 @@
+class MainSearch extends SearchForm {
+  constructor() {
+    super();
+    this.allSearchInputs = document.querySelectorAll('input[type="search"]');
+    this.setupEventListeners();
+  }
+
+  setupEventListeners() {
+    let allSearchForms = [];
+    this.allSearchInputs.forEach(input => allSearchForms.push(input.form))
+    this.input.addEventListener('focus', this.onInputFocus.bind(this));
+    if (allSearchForms.length < 2) return;
+    allSearchForms.forEach(form =>
+      form.addEventListener('reset', this.onFormReset.bind(this))
+    );
+    this.allSearchInputs.forEach(input =>
+      input.addEventListener('input', this.onInput.bind(this))
+    );
+  }
+
+  onFormReset(event) {
+    super.onFormReset(event);
+    if (super.shouldResetForm()) {
+      this.keepInSync('', this.input);
+    }
+  }
+
+  onInput(event) {
+    const target = event.target;
+    this.keepInSync(target.value, target);
+  }
+
+  onInputFocus() {
+    const isSmallScreen = window.innerWidth < 750;
+    if (isSmallScreen) {
+      this.scrollIntoView({behavior: 'smooth'});
+    }
+  }
+
+  keepInSync(value, target) {
+    this.allSearchInputs.forEach(input => {
+      if (input !== target) {
+        input.value = value;
+      }
+    });
+  }
```

Comment:
One small thing with the sync but I don't think of it as a blocker: [video](https://screenshot.click/12-01-i30oe-hbv29.mp4)

Reply:
I did some changes to fix that, take a look at my last commit and tell me what do you think!